src/chrdev.rs

```diff
@@ -144,12 +167,31 @@ unsafe extern "C" fn release_callback<T: FileOperations>(
     return 0;
 }
 
+unsafe extern "C" fn llseek_callback<T: FileOperations>(
+    file: *mut bindings::file,
+    offset: bindings::loff_t,
+    whence: c_types::c_int,
+) -> bindings::loff_t {
+    let off = match whence as u32 {
+        bindings::SEEK_SET => SeekFrom::Start(offset as u64),
```

Comment:
Is `as u64` the right thing here? (What does native C code do?)

Reply:
Unclear! The values are all i64s -- this is the type rust itself uses for this type. a negative value for offset from start is nonsense?